### PR TITLE
Static/plants/108 division

### PR DESCRIPTION
**Pull request by gnaamati**
Division updated for 108 in Plants

### CODE REVIEW

File: plants/whatsnew.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-    - [Cowpea](/Vigna_unguiculata) (_Vigna unguiculata_) : assembly from ENA [GCA_004118075.1](https://www.ebi.ac.uk/ena/browser/view/GCA_004118075.1) and annotation from [JGI](https://phytozome-next.jgi.doe.gov/cowpeapan/info/Vunguiculata_v1_2). 
-    - [Barnyard Grass](/Echinochloa_crusgalli) (_Echinochloa crusgalli_) : assembly and from NCBI [GCA_020466025.1](https://www.ncbi.nlm.nih.gov/assembly/GCA_020466025.1) and annotation from [ZJU](http://ibi.zju.edu.cn/RiceWeedomes/Echinochloa/).
+    - [Ryegrass](/Lolium_perenne) (_Lolium perenne_) : assembly from ENA [GCA_019359855.1](https://www.ncbi.nlm.nih.gov/assembly/GCA_019359855.1) and annotation from [Cyverse](https://datacommons.cyverse.org/browse/iplant/home/shared/commons_repo/curated/Copetti_Kyuss_assembly_annotation_March_2021).
+    - [Brown Mustard](/Brassica_juncea) (_Brassica juncea_) : assembly from ENA [GCA_018703725.1](https://www.ncbi.nlm.nih.gov/assembly/GCA_018703725.1) and annotation from [oilseedhunan](http://www.oilseedhunan.net/download.html).
+    - [Cork Oak](/) (_Echinochloa crusgalli_) :  assembly and annotation from NCBI [GCA_002906115.4](https://www.ncbi.nlm.nih.gov/assembly/GCA_002906115.4).
```
Comment (msmanoj) [inline plants/whatsnew.md:6]: @gnaamati  - can you please check if this line is correct and has the correct URL set?

Comment (gnaamati) [reply plants/whatsnew.md:6]: Hi @msmanoj, which line are you referring to?

Comment (gnaamati) [reply plants/whatsnew.md]: I've updated the file to remove one species

Comment (msmanoj) [reply plants/whatsnew.md]: It looks like `Cork Oak` is missing the url after these changes. Is that expected?

**Pull request merged**